scripts-for-reference/run-deepforest-prediction-from-command-line.py

```diff
@@ -1,6 +1,9 @@
-# This script can be called from the command line. It is a wrapper for several functions in DeepForest (primarily predict_tile and bboxes_to_shapefile).
-# This streamlines calling DeepForest from R, expecially calling it multiple times with different parameters.
-# The structure of the command line call is: `python3 run-deepforest-prediction-from-command-line.py {orthomosaic file path} {window size} {output bounding box gpkg file path}`
+# Title: Command Line Deepforest Prediction
+# Description: This script streamlines calling DeepForest from R, expecially
+#              calling it multiple times with different parameters. The script can
+#              be called from the command line and is a wrapper for several functions
+#              in DeepForest (primarily predict_tile and bboxes_to_shapefile).
+# Credit: Largely based on a script developed by Derek Young
```

Comment:
Your script headers are great! Could you please add the new command line call structure, similar to what was previously in the header:

```
# The structure of the command line call is: `python3 run-deepforest-prediction-from-command-line.py {orthomosaic file path} {window size} {output bounding box gpkg file path}
```